### DISCUSSION TITLE
Commercial Use License

### DISCUSSION THREAD

**Discussion by designnatix** [carbon-design-system/carbon#7657] [Help] [answered]
We have some questions about the license for Carbon Design System. As we are a commercial project, is it possible to use the Carbon Design System for our commercial project? 

I did find this license page on GitHub, and it seems to suggest that the Carbon Design System is open source under Apache 2.0. Can you please confirm that is true? We would branch off the Carbon Design System and make our own changes and customization.

https://github.com/carbon-design-system/carbon/blob/master/LICENSE

We would also like to use the code in addition to the Design System for our commercial project. Is that OK?

**Comment by joshblack** [comment, marked as answer]
Hi there @designnatix! 👋 

To confirm, the code for Carbon is licensed under the Apache-2.0 license. This means that you can totally use it for commercial use, modify it, etc. but there are limitations in terms of trademark, liability/warranty.

![image](https://user-images.githubusercontent.com/3901764/106063020-32a9b400-60bd-11eb-8497-11b228f9b4ba.png)

**Reply by designnatix** [reply]
Thank you, Josh! How about the design system? Is that also under an Apache 2.0 license?

**Reply by jnm2377** [reply]
Hey @designnatix when you say "use the code in addition to the design system", do you mean using the components and customizing them, or do you mean copying the actual source code and making changes there?

**Reply by designnatix** [reply]
Hello! We would like to use both the design system and the React and other javascript libraries in our product. We would branch from your repo and make customizations. All of this would be for a commercial project—so we would like to understand if your license allows commercial use. 
Thanks

**Reply by joshblack** [reply]
Great questions @designnatix, yes the Apache-2.0 license includes commercial use and is applied to the code that is used to build the design system. That means that companies are free to fork and modify [this repo](https://github.com/carbon-design-system/carbon) and use that code for commercial use.

I believe the only limitations that the license specifies is that companies can't make it seem they themselves are Carbon, are providing Carbon resources, etc.

If you all are looking to develop a custom theme, you can also totally use Carbon as-is and create your theme on top of it. We're more than happy to help out with those use-cases and make it easier for teams to build on top of Carbon and customize where needed 😄

**Reply by designnatix** [reply]
Thank you @joshblack! That is great. We appreciate your offer to help and will be in touch once we are getting to start. All the best!
